reliability.md

```diff
@@ -1,33 +1,49 @@
 
-It's recommended to keep and test regular backups of your WordPress sites using the system-level backup or snapshot infrastructure of your choice. One thing to be aware of is that the code, assets and database change in conjunction with WordPress but may be backed up separately. For this reason, it is a good idea to keep restore points that include backups of code, assets, and database taken at the same point in time. It’s recommended to create restore points before any critical action, e.g. WordPress core update.
+When you make a backup you have to do it from those three elements, since one does not work without the others.
 
-## Monitoring
+### Where to make copies
+
+There are many ways to back up, so you must use the best for your project / site.
+
+The first one is the hosting server itself. Usually providers themselves offer the possibility of making a copy of all the contents hosted on the machine. Usually these copies are made externally on a specific backup server. In this case, you should know how it works and when the copies are done, as well as how to restore the copy, if it can be partial or total, and if you have access to the data in a simple way.
+
+A second way is manually, through the operating system. In this case you must create everything to make a copy of all files (for example in a ZIP), dump the database to obtain a copy of the SQL file with all the data and also the configurations. Once you have all the information, you should try to have a local copy and an external out the actual network.
+
+WordPress users may not know how they can access the server o create a manual backup, so the easiest way is to use a backup plugin. There are many [backup plugins](https://wordpress.org/plugins/search/backup/) and you should try them until you find the one that suits your needs. It is not the same to make a backup of a small site (500 MB) than a large one (5 GB). Response times and server's performance could cause saturation on the system and not create the copies; try different plugins until you find the one that works the best.
 
-Site monitoring systems and services can notify you when your site isn't working properly. They can often correct any minor issues, or help you to do so before they become major issues.
+It is also important to have redundancy on your copies. This means you should have a local copy (for easy access in simple restoring) for that moment when your site gets corrupted but it doesn't affect the machine, so having the files close by and available quickly is usually very helpful. The other is to have them outside the server, in an external place, even outside the same network or provider or country. The fact that it is in another place would allow that even if that provider had a serious problem (and for example you had no possibility to access it) you could restore your site from an external place or another provider.
 
```

Comment:
I think something along the lines of practice a full restore from your backup is needed.

Reply:
Another thought of database backups, describe pros/cons of physical and logical backups. Point in time recovery is another aspect to look for in backup plugins if a requirement.